packages/datepicker/src/react/context.ts

```diff
@@ -1,6 +1,16 @@
 import React from 'react'
 import type { Calendar } from 'dayzed'
-
+interface DateContextInterface extends Calendar {
+  'aria-labelledby': string
+  dayKeyHandlers: (
+    date: Date,
+    dateObj: {
+      [x: string]: any
+    }
+  ) => React.KeyboardEventHandler<HTMLButtonElement>
+}
 const initialValue = {}
-// @ts-ignore: really hate this part of context and typescript
-export const DateContext = React.createContext<Calendar>(initialValue)
+export const DateContext = React.createContext<DateContextInterface>(
+  // @ts-ignore: really hate this part of context and typescript
```

Comment:
This seems like a bad idea. Above you have the interface showing non-null types. Then here we suppress the error. If the fields here are potentially nullable, TS should know and then we will too when we go to code against context fields. 

What do you think? Would love to hear counter arguments, if you prefer.